app/assets/stylesheets/app/uploads.scss

```diff
@@ -15,12 +15,16 @@
   -webkit-user-select: none;
   -ms-user-select: none;
 
+  &.input-group > :not(:first-child):not(.dropdown-menu):not(.valid-tooltip):not(.valid-feedback):not(.invalid-tooltip):not(.invalid-feedback)  {
```

Comment:
Yes. OOF on this selector. Since the pause button is technically the second rendered item, bootstrap wanted to take away a 1px margin. that caused a jump when toggling the pause button.